skeleton/cli-bundler/aurelia_project/tasks__if_typescript/transpile.ts

```diff
@@ -50,9 +52,21 @@ export function buildPluginJavaScript(dest, format) {
       typescript: require('typescript'),
       module: format
     });
+
+    let defaultBuildOptions = {
+      minify: 'stage & prod',
+      sourcemaps: 'dev & stage',
+      rev: false
+    };
+
+    let buildOptions = new Configuration(project.build.options, defaultBuildOptions);
+    let sourcemaps : boolean = buildOptions.isApplicable('sourcemaps');
+
     return gulp.src(project.transpiler.dtsSource)
       .pipe(gulp.src(project.plugin.source.js))
+      .pipe(gulpIf(sourcemaps, gulpSourcemaps.init()))
       .pipe(typescriptCompiler())
+      .pipe(gulpIf(sourcemaps, gulpSourcemaps.write('.', { includeContent: false, sourceRoot: '../../src' })))
```

Comment:
Following code worked for me. BTW you only touched transpile.ts for TS, not transpile.js for babel. You need both for the PR to be complete.

```js
      .pipe(gulp.src(project.plugin.source.js, {sourcemaps: true})
      .pipe(transpile) // or typescriptCompiler()
      .pipe(gulp.dest(dest, {sourcemaps: true}));
```